Dockerfile

```diff
@@ -1,22 +1,22 @@
 #
 # Dockerfile for Storm Mesos framework
 #
-FROM mesosphere/mesos:0.22.0-1.0.ubuntu1404
+FROM mesosphere/mesos:0.25.0-0.2.70.ubuntu1404
 MAINTAINER Timothy Chen <tnachen@gmail.com>
 
 # build packages
 ENV DEBIAN_FRONTEND noninteractive
 RUN apt-get update
-RUN apt-get install -yq openjdk-7-jdk
+RUN apt-get install -yq openjdk-7-jdk maven wget
 
 # export environment
 ENV JAVA_HOME /usr/lib/jvm/java-7-openjdk-amd64
 
 ENV MESOS_NATIVE_JAVA_LIBRARY /usr/lib/libmesos.so
 
-# copy local checkout into /opt/storm
-ADD . /opt/storm
+ADD . /work
 
+WORKDIR /work
+RUN ./bin/build-release.sh main
```

Comment:
@brndnmtthws : ok, so I'm getting back to testing with the docker stuff in this repo -- it seems your changes have totally busted the `bin/build-release.sh dockerImage` option -- so I'm guessing you didn't notice that option and are just running the docker build manually.  I did it like so:
```
docker build -t mesos/storm:git-`git rev-parse --short HEAD` .
```

Something I don't like about this new style of buliding docker images is that it isn't caching any of the java artifacts being downloaded, so it makes rebuilds slow.  Any ideas for improving that behavior?